### PR TITLE
Dcwither/option update

### PR DESCRIPTION
**Pull request by dcwither**
Addresses comments from #18. Currently untested, so not in a good state to merge.

### CODE REVIEW

**Comment by dcwither** [comment]
I've managed to fix a few bugs, leading to a passing run, though unfortunately I've been unable to test this thoroughly within this repo. 

The check fails here due to permissions, and it simply doesn't run when I attempt to merge to my master regardless of changing the `on` configuration to any pull request. You might be able to rerun this to test it out, though I encourage you to pull it down and take a look.

I'll also pursue testing this branch running on another project when I can find the time.

**Comment by andresz1** [comment]
Hi @dcwither! looks nice! 🎉   thank you so much! I think that you can test it creating a PR to your own forked repo, have you tried doing that ?. Sorry for late reply 😞  I've been really busy this past two weeks I'll try this out ASAP.

Thank you again and let me know if you can test it that way 😉

**Comment by dcwither** [comment]
>  I think that you can test it creating a PR to your own forked repo, have you tried doing that? 

I did try that, I'm not sure why it didn't run at all. Here's the links to the attempts (though they're closed so they're not showing the checks) https://github.com/chanzuckerberg/size-limit-action/pull/3 https://github.com/chanzuckerberg/size-limit-action/pull/2

> Sorry for the late reply 😞 I've been really busy this past two weeks I'll try this out ASAP.

No worries, I've been busy as well and haven't had the opportunity to test this branch with another project that would hopefully confirm things. Thanks for taking a look when you can 😄

File: src/main.ts [hunk 1]
```diff
@@ -1,11 +1,42 @@
...
+  COMMENTED: "COMMENT",
+  CHANGES_REQUESTED: "REQUEST_CHANGES"
+};
+
+async function fetchPreviousReview(
+  octokit: GitHub,
+  repo: { owner: string; repo: string },
+  pr: { number: number }
+) {
+  const reviews: Octokit.PullsListReviewsResponse = await octokit.paginate(
```
Comment (andresz1) [inline src/main.ts:23]: Nice! this will return all reviews right ? since `list*` only return the first 30

**Comment by andresz1** [comment]
Hi @dcwither! I thought that maybe making you collaborator will enable you to test it out but now I think that just forking the repo yourself and not from an orga would work. Can you try it and let me know if it works (see forked repo [PR](https://github.com/vincentmacri/size-limit-action/pull/1))

**Pull request merged**